Makefile

```diff
@@ -70,7 +71,7 @@ test_cpython_doctest:
 	${PY} -m pytest pyboy/ -n auto -v
 
 test_cython:
-	${PY} -m pytest tests/ -n auto -v
+	${PY} -m pytest tests/ ${TEST_CYTHON_ARGS}
 
 test_pypy:
 	${PYPY} -m pytest tests/ pyboy/ -n auto -v
```

Comment:
We could call them `${PYTEST_ARGS}` and add them to PyPy too. They would normally run the same setup.